frontend/components/nav_bar/nav_bar.tsx

```diff
@@ -0,0 +1,43 @@
+import Text from "@components/text/text";
+import classNames from "classnames";
+import Image from "next/image";
+import Link from "next/link";
+import styles from "./nav_bar.module.scss";
+
+type NavBarProps = {
+  className?: string;
+};
+
+const NavBar = ({ className }: NavBarProps) => {
+  return (
+    <div className={classNames(styles.nav_bar, className)}>
+      <Image
+        className={styles.image}
+        src="/rtp_logo.png"
```

Comment:
Would be nice for this image to be either configurable as a prop or the hard coded data stored somewhere else eg in a nav_bar_static_content.ts file

Reply:
I think the latter would make more sense. I can do that